components/mcb/Core/Inc/mcb.h

```diff
@@ -61,9 +61,8 @@
  */
 #define ADC_LOWER_DEADZONE            900
 #define ADC_FOR_NO_SPIN               1300    
-#define ADC_MIN_FOR_FULL_THROTTLE     1830
-#define ADC_MAX_FOR_FULL_THROTTLE     2000
-
+#define ADC_MIN_FOR_FULL_THROTTLE     2000
+#define ADC_MAX_FOR_FULL_THROTTLE     2600
```

Comment:
What experimentation led to these changes?

Reply:
On Thursday, we found that the ADC out of range fault would be triggered when Diego pressed the pedal all the way. We recalibrated theses values using the mcb diagnostics message.

Reply:
Got it. Thanks